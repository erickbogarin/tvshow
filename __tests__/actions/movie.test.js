import * as Actions from 'actions';

test('findMovie should return an action', () => {
  expect(Actions.findMovie({})).toMatchSnapshot();
});

test('resetMovie should return an action', () => {
  expect(Actions.resetMovie({})).toMatchSnapshot();
});
