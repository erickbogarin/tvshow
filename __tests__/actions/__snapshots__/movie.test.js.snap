exports[`test resetSearch should return an action 1`] = `
Object {
  "type": "RESET_SEARCH",
}
`;

exports[`test setMovies should return an action 1`] = `
Object {
  "payload": Object {},
  "type": "SET_MOVIES",
}
`;
